src/components/index.ts

```diff
@@ -13,6 +13,9 @@ export { default as Center } from './center';
 export { default as Checkbox } from './checkbox';
 export { default as DatePicker } from './datePicker';
 export { default as DevOverlay } from './devOverlay';
+export { default as Drawer } from './drawer';
+export { default as DrawerContent } from './drawer/DrawerContent';
+export { default as DrawerOverlay } from './drawer/DrawerOverlay';
```

Comment:
Why don't we choose to build and export the "full" component here (like the one you display in the story) instead of having to recreate it with these 3 components? Genuinely asking, I may have missed something 🙈

Reply:
yea I made it as a standalone component to have ability to reuse it, and in the story it's just an example of usage. But it might make sense to create separate collapsible component with a drawer exactly for header mega menu